src/backend/gl/src/command.rs

```diff
@@ -733,21 +740,27 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
                 error!("Number of viewports can not be zero.");
                 self.cache.error_state = true;
             }
-            n if n <= self.limits.max_viewports => {
-                self.push_cmd(Command::SetViewports { viewport_ptr, depth_range_ptr });
+            n if n <= self.limits.max_viewports - (first_viewport as usize) => {
```

Comment:
`if n + first_viewport <= max_viewports` nicer imo. Could remove the check at the beginning then.